packages/eslint-plugin-pf-codemods/lib/rules/createContainer-insteadof-allowZoom.js

```diff
@@ -0,0 +1,70 @@
+const { renameProps0, getPackageImports } = require('../helpers');
+
+// https://github.com/patternfly/patternfly-react/pull/4278
+const renames = {
+  'Chart': {
+    allowZoom: 'containerComponent={<VictoryZoomContainer />}'
+  },
+  'ChartGroup': {
+    allowZoom: ''
+  }
+};
+
+const ensurePackage = 'victory-zoom-container';
+const compMap =[ 'VictoryZoomContainer' ];
+
+module.exports = {
+  meta: {
+    messages: {
+      missingImportMsg: "`add missing imports {{ missingImports }} from {{ ensurePackage }}",
+    },
+    fixable: "code",
+    schema: []
+  },
+  create: function(context) {
+      const imports = getPackageImports(context, '@patternfly/react-charts');
+      return imports.length === 0 ? {} : {
+        JSXOpeningElement(node) {
+          renameProps0(context, imports, node, renames);
+        },
+        ImportDeclaration(node) {
+          const packageImports = getPackageImports(context, ensurePackage);
+          const packageImportNames = packageImports.map(imp => imp.imported.name);
+          const myImports = node.specifiers.map(imp => imp.imported.name);
+          const missingImports = compMap
+              .filter(imp => !packageImportNames.includes(imp)) // not added by consumer
+              .filter(imp => !myImports.includes(imp)) // not added by this rule
+              .join(', ');
+
+          if (packageImports.length == 0 ) {
```

Comment:
Why not keep the code DRY and this `if (packageImports.length == 0 )` logic to the helper?

Reply:
I could do this I was unsure if we wanted to populate the helper with it, thats all. I will move it over now and push it up.

Reply:
Thank you!

Reply:
Actually the main reason I did not do this is that the first check in the helper function is never going to allow the code to trigger.

  if (node.source.value !== package) {
    return;
  }

In this case the import is not tied to a node as it is being used inside an attribute.